x-pack/metricbeat/module/mssql/_meta/docs.asciidoc

```diff
@@ -8,8 +8,12 @@ The module is being tested with https://hub.docker.com/r/microsoft/mssql-server-
 [float]
 == Metricsets
 
-The following Metricset is already included:
+The following Metricsets are already included:
 
-=== `db`
+=== `log`
```

Comment:
Sounds like next we need the MSSQL module for the actual logs :-)

Reply:
Renamed to `transaction_log`